packages/content-conformance/src/rules/top-of-file-h1.js

```diff
@@ -0,0 +1,31 @@
+// eslint-disable-next-line import/no-anonymous-default-export
+/** @type {import('../types.js').ConformanceRuleBase} */
+export default {
+  type: 'content',
+  id: 'top-of-file-h1',
+  description: 'Each content file must contain an H1 the top.',
+  executor: {
+    async contentFile(file, context) {
+      file.visit(['root'], (node) => {
+        const firstNode = node.children[0]
+        const contentFileName = file.history[0].split('/').pop()
```

Comment:
[question]: Can we simplify this to use [`file.path`](https://github.com/vfile/vfile#filepath)?

[suggestion]: Using `file.path` (ex. `foo/bar/index.mdx`) could be useful here for clarity. 
We have multiple `.../index.mdx` files. So if we're only surfacing the basename, and if several of the same named files are violating this rule, it could be tedious to find the correct one(s).

Reply:
Sounds like a good idea!